messaging.rst

```diff
@@ -50,7 +54,7 @@ Fields
 Merkle Tree
 -----------
 
-A binary tree composed of the hash of the locks. The root of the tree is the value used in the :ref:`balance proofs <balance-proof-message>`. The tree is changed by the ``LockedTransfer``, ``RemoveExpiredLock`` and ``Unlock`` message types.
+A binary tree composed of the hash of the locks. The root of the tree is the value used in the :ref:`onchain balance proofs <balance-proof-onchain>`. The tree is changed by the ``LockedTransfer``, ``RemoveExpiredLock`` and ``Unlock`` message types.
 
```

Comment:
I would mainly reference the `offchain` balance proof here, because this is the one used in the client in conjunction with the merkle tree. The reference to the `onchain` one (in the offchain balance proof definition) should be enough to make the connection with the contracts.

Reply:
I'll change the link to terminology::Balance Proofs.

Reply:
Done.